projects/storefrontstyles/scss/components/form-components/_star-rating.scss

```diff
@@ -1,19 +1,26 @@
 %cx-star-rating {
-  .star {
-    position: var(--cx-position, relative);
-    display: var(--cx-display, inline-block);
-    color: var(--cx-color, var(--cx-g-color-light));
-    margin: var(--cx-margin, 0 5px 0 0);
-  }
+  padding: 15px 0;
 
-  .full {
-    color: var(--cx-color, var(--cx-g-color-primary));
+  &:not([disabled='true']) {
+    cursor: pointer;
   }
 
-  .half {
-    position: var(--cx-position, absolute);
-    display: var(--cx-display, inline-block);
-    overflow: var(--cx-overflow, hidden);
-    color: var(--cx-color, var(--cx-g-color-primary));
+  .star {
+    padding-right: 5px;
+    font-size: 20px;
```

Comment:
This is a personal thing. But maybe to preserve the old styling this could be smaller.

Reply:
I've done this explicitly to address #2249